crates/swc_ecma_transforms_typescript/src/strip.rs

```diff
@@ -18,7 +18,7 @@ use swc_ecma_utils::{
 use swc_ecma_visit::{as_folder, Fold, Visit, VisitMut, VisitMutWith, VisitWith};
 
 /// Value does not contain TsLit::Bool
-type EnumValues = AHashMap<Id, TsLit>;
+type EnumValues = AHashMap<JsWord, TsLit>;
```

Comment:
I think it's ok to use `JsWord` here. It seems typescript itself doesn't really handle enums nested within enums that gracefully (because who writes code like that?)

Reply:
You are right.